websites/W/Wolfy/presence.ts

```diff
@@ -54,63 +160,75 @@ async function addJoinGameButton(presenceData: PresenceData, gameId: string) {
 		presenceData,
 		`(${
 			document.querySelector("div.Header_timer__36MsP")?.textContent
-		}) Rejoindre la partie`,
+		}) ${getString("joinGame")}`,
 		`https://wolfy.fr/game/${gameId}`
 	);
 }
 
 function addVisitHelpCenterButton(presenceData: PresenceData) {
 	addButton(
 		presenceData,
-		"Consulter le centre d'aide",
+		getString("visitHelpCenter"),
 		"https://help.wolfy.fr"
 	);
 }
 
+function addVisitEventButton(presenceData: PresenceData, href: string) {
+	addButton(presenceData, getString("toConsultEvent"), href);
+}
+
 async function addVisitProfilButton(
 	presenceData: PresenceData,
 	username: string
 ) {
 	if (!username) return;
 	if (!(await presence.getSetting("visitProfileButton"))) return;
 
-	let label = `Visiter le profil de ${username}`;
+	let label = `${getString("visitProfile").replace("$username", username)}`;
 	if (label.length > 32) label = `${label.slice(0, 31)}…`;
 
 	addButton(presenceData, label, `https://wolfy.fr/leaderboard/${username}`);
 }
 
 function addVisitWolfyButton(presenceData: PresenceData) {
-	addButton(presenceData, "Visiter le site de Wolfy", "https://wolfy.fr");
+	addButton(presenceData, getString("visitWolfy"), "https://wolfy.net");
```

Comment:
Should probably check for the user's language and redirect them to the appropriate website or use document.location.origin

Reply:
Wolfy is not officially Wolfy.net, wolfy.fr always redirect to Wolfy.net

Reply:
Also it use to always link to wolfy.net even thought we are browsing help.wolfy.net